LLMonFHIR/FHIRInterpretation/UserStudy/Survey/Views/TaskInstructionView.swift

```diff
@@ -0,0 +1,44 @@
+//
+// This source file is part of the Stanford Spezi project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+struct TaskInstructionView: View {
```

Comment:
It seems like the instructions view doesn't scale great with increased text sizes, we might either want to add an scrollview here and also expand it larger if we have larger text sizes?
![RocketSim_Screenshot_iPhone_16_Pro_6 3_2025-05-05_16 47 05](https://github.com/user-attachments/assets/7bd82887-25b0-4449-ae88-5fa75ea5b360)

Reply:
The sheet’s height is currently fixed relative to the screen. I initially used a List, but it wasn't convinced. The view is scrollable even when there’s enough space to display all text, and reading while scrolling in this small view wasn’t ideal. 
In this case, I like more the approach of making the sheet to adapt its height based on the text size.

Reply:
https://github.com/user-attachments/assets/7053c383-d261-4e20-b7ad-15da859040b7

Reply:
Nice, looks great!! 🚀